artiq/language/environment.py

```diff
@@ -54,6 +54,7 @@ class PYONValue(_SimpleArgProcessor):
     def __init__(self, default=NoDefault):
         # Override the _SimpleArgProcessor init, as list defaults are valid
         # PYON values
+        # default stays decoded
```

Comment:
What does that mean?

Reply:
I was a bit confused by the `process`ing logic for `PYONValue`. For all other argument processors, the default value is stored processed, except for `PYONValue`. The client code seems to expect that `process(val)` and `default()` both return pyon-decoded values.
But I can remove the comment.